sdk/attestation/azure-security-attestation/src/attestation_client.cpp

```diff
@@ -285,9 +285,6 @@ void AttestationClient::RetrieveResponseValidationCollateral(Azure::Core::Contex
 
 /** @brief Construct a new Attestation Client object
  *
- * @param endpoint The URL address where the client will send the requests to.
- * @param credential The authentication method to use (required for TPM attestation).
- * @param options The options to customize the client behavior.
  */
```

Comment:
Same here/elsewhere. Remove duplicate comments in source file. We don't want to maintain 2 places for the same thing that might start differing subtly.